WordPress/Docs/CodeAnalysis/PerfomantPHPFunctionsStandard.xml

```diff
@@ -0,0 +1,22 @@
+<documentation title="Prefer Performant PHP">
+    <standard>
+        <![CDATA[
+    Various PHP functions are legacy and less performant than other language constructs.
```

Comment:
These functions aren't necessarily legacy, but should basically only be used in callbacks.

The "standard" description as you have it now is also more generic than what the sniff actually does.